scanner/cmd/updater/main.go

```diff
@@ -16,7 +16,7 @@ import (
 const DefaultURL = "https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml"
 
 func tryExport(ctx context.Context, outputDir string, opts *updater.ExportOptions) error {
-	const timeout = 2 * time.Hour
+	const timeout = 5 * time.Hour
```

Comment:
IIRC, the updater GHA job runs every 4 hours. If that's the case, what are the implications of making this timeout greater than that value?